### ISSUE TITLE
jellyfish.soundex('a') != cjellyfish.soundex('a')

### ISSUE THREAD

**Issue by nchammas** [jamesturk/jellyfish#65]
This could be a bug, but it could also just be a matter of differing interpretations on what "correct" is.

Here's the repro in Python 3.5:

```python
>>> import jellyfish as cjellyfish
>>> import jellyfish._jellyfish as jellyfish
>>> cjellyfish.soundex('a')
'A000'
>>> jellyfish.soundex('a')
'a000'
>>> jellyfish.soundex('a') == cjellyfish.soundex('a')
False
```

Obviously, it's easy to just `upper()` the output of each function before comparing them, but there is a technical issue: `soundex()` is returning different output depending on whether the Python or C implementation is used. Ideally, the only difference between the behavior of the two implementations should be their performance.

I raise this issue because I'm trying to implement property-based tests for Jellyfish. These tests will make it much easier for us to ensure that the Python and C implementations of each function are returning identical output, without needing to manually write up dozens of test cases. Before diving too deep into this, I want to make sure I'm working with the same idea of "correct behavior" that you see for these functions.

Do you think the above behavior is a bug?

**Comment by jamesturk** [comment]
Ah yeah that's definitely a bug, nice catch!

**Comment by nchammas** [comment]
Fixed by #69.
